block/manager.go

```diff
@@ -496,6 +494,14 @@ func (m *Manager) publishBlock(ctx context.Context) error {
 		// set the commit to current block's signed header
 		block.SignedHeader.Commit = *commit
 
+		// set the validator set using the signer's public key
+		// TODO(ganesh): need to hook into a module that selects signers
```

Comment:
any better way to handle this? @nashqueue @S1nus ?

Reply:
i think we should take this up in the next issue. both signers and signatures are to be handled in the staking module.

Reply:
https://github.com/rollkit/rollkit/issues/797